packages/app-page-builder/src/editor/plugins/elementSettings/variable/RichVariableInput.tsx

```diff
@@ -1,23 +1,64 @@
-import React, { useState } from "react";
+import React, { useCallback, useState } from "react";
 import styled from "@emotion/styled";
 import { CoreOptions } from "medium-editor";
+import { ReactComponent as ExpandIcon } from "~/admin/assets/expand.svg";
```

Comment:
Please use the `@material-design-icons/svg` package for icons. It contains all icons from https://fonts.google.com/icons?selected=Material+Icons so we don't have to manually download SVGs each time, plus it saves on the bundle size. I discovered that package a few months ago and we're now switching to that across the entire admin app.